nemo/utils/exp_manager.py

```diff
@@ -501,6 +504,11 @@ def exp_manager(trainer: 'pytorch_lightning.Trainer', cfg: Optional[Union[DictCo
         # Add lightning file logging to global_rank zero
         add_filehandlers_to_pl_logger(log_dir / 'lightning_logs.txt', log_dir / 'nemo_error_log.txt')
 
+    elif trainer.num_devices * trainer.num_devices > 1:
```

Comment:
Was it meant to be `trainer.num_nodes * trainer.num_devices`?

Reply:
ah thanks for catching that!

Reply:
yeah it's not a big deal though since if it's unlikely that someone runs with `num_nodes > 1` and `num_devices == 1` ;)